app/Console/Commands/SendNotificationForOpenCFPs.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+namespace App\Console\Commands;
+
+use App\User;
+use App\Conference;
+use Illuminate\Console\Command;
+use App\Notifications\CFPIsOpen;
+
+class SendNotificationForOpenCFPs extends Command
+{
+    /**
+     * The name and signature of the console command.
+     *
+     * @var string
+     */
+    protected $signature = 'symposium:notifyCfps';
+
+    /**
+     * The console command description.
+     *
+     * @var string
+     */
+    protected $description = 'Send out notification for open CFPs';
+
+    /**
+     * Execute the console command.
+     *
+     * @return mixed
+     */
+    public function handle()
+    {
+        Conference::Approved()->NotShared()->UnclosedCfp()->each(function ($conference) {
```

Comment:
I think I prefer starting scopes with a lowercase letter, so:

```php
Conference::approved()->notShared()->unClosedCfp()->each(
```

Reply:
Also, I think the unClosedCfp scope will include those which have not opened yet, either. What you likely want is the `openCfp` scope.